src/transformers/pipelines/__init__.py

```diff
@@ -384,6 +386,13 @@
         "default": {"model": {"pt": ("MCG-NJU/videomae-base-finetuned-kinetics", "4800870")}},
         "type": "video",
     },
+    "mask-generation": {
+        "impl": MaskGenerationPipeline,
+        "tf": (),
+        "pt": (AutoModelForMaskGeneration,) if is_torch_available() else (),
+        "default": {"model": {"pt": ("facebook/sam-vit-huge", "")}},
```

Comment:
Revision is mandatory, so please put one!

Reply:
Fixed in[b9e4272](https://github.com/huggingface/transformers/pull/22840/commits/b9e4272b67f93798ee1f3c3e5eeb70b567e2bf9b)